tools/.lintstagedrc.json

```
@@ -6,8 +6,8 @@
   "../**/*.dart|*.dart": [
     "flutter format"
   ],
-  "../**/*.tf|*.tf": [
-    "terraform fmt"
+  "../": [
```

I just ran it in the top level folder and it seemed to take a second to run. A little annoying to add that cost to everyone. I think this only runs it when there's a change that overlaps with the terraform directory:
```suggestion
  "../server/terraform/**": [
```